From 056b1d368b14e9664b69499cbcba49bc04be7e52 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Thu, 21 Apr 2022 11:11:22 +0200
Subject: [PATCH] hwdb: Add Greaseweazle "drives" to the list of analyzers

They're floppy disk flux readers and writers used in digital
preservation and can be broadly considered to be "analyzers" of magnetic
fluxes.

This will have the intended side-effect of giving access to the device
to users at the console, obsoleting:
https://github.com/keirf/greaseweazle/blob/master/scripts/49-greaseweazle.rules

(cherry picked from commit 9ab755dce50bfb6f1f614f57fe3a8b65b570b5b9)

Related: #2087778
---
 hwdb.d/70-analyzers.hwdb | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/hwdb.d/70-analyzers.hwdb b/hwdb.d/70-analyzers.hwdb
index 7f31ca3ea3..821ebcb1ca 100644
--- a/hwdb.d/70-analyzers.hwdb
+++ b/hwdb.d/70-analyzers.hwdb
@@ -9,6 +9,13 @@
 #   Specify if a device is a signal analyzer
 #   ID_SIGNAL_ANALYZER=1|0
 
+###########################################################
+# Greaseweazle
+###########################################################
+# Greaseweazle
+usb:v1209p4D69*
+ ID_SIGNAL_ANALYZER=1
+
 ###########################################################
 # Total Phase
 ###########################################################
